middleware/queue/source/group/main.cpp

```diff
@@ -60,6 +63,24 @@ namespace casual
 
                } // select
 
+               namespace sqlite::library::version
+               {
+                  constexpr int sqlite_version_number( int major, int minor, int patch)
+                  {
+                     return major * 1000000 + minor * 1000 + patch;
+                  }
```

Comment:
Put this in _sql::database_ ? It feels like it belongs there. We might need to constrain for other versions in other managers later on.

Reply:
Aye